lib/Runtime/Language/SourceTextModuleRecord.cpp

```diff
@@ -175,6 +175,12 @@ namespace Js
             }
 
             OUTPUT_TRACE_DEBUGONLY(Js::ModulePhase, _u("\t>NotifyParentAsNeeded\n"));
+            if (this->promise != nullptr)
+            {
+                SourceTextModuleRecord::ResolveOrRejectDynamicImportPromise(false, this->GetErrorObject(), this->GetScriptContext(), this);
```

Comment:
>this->GetScriptContext() [](start = 107, length = 24)

nitpick : use the local scriptContext (also you can use the errorObject as well)